### PR TITLE
Feature top page implementation

### PR DESCRIPTION
**Pull request by curtisahuang**
## Description

Making the top page for logged in and logged out users

## Closes issue(s)

If there any issue that is fixed with this PR, please attach it/them here...

## Screenshots

If you have image share...

## Changes include

- [ ] Bugfix (non-breaking change that solves an issue)
- [X] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

## Checklist

- [ ] I have tested this code
- [ ] Add here the checks for this PR. If any please delete these checks

### CODE REVIEW

File: src/components/SignIn/index.tsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import StyledFirebaseAuth from "react-firebaseui/StyledFirebaseAuth";
 import firebase from "../../utils/firebase";
 
 const uiConfig = {
-  signInFlow: "popup",
+  signInFlow: "redirect",
   signInSuccessUrl: "/user/myGardens",
```
Comment (daiHash) [inline src/components/SignIn/index.tsx:7]: 👍

File: src/pages/About/index.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+import "./About.css";
+import { Paper } from "@material-ui/core";
+
+export const About = () => {
+  const { userData } = useUserState();
+  
+  let history = useHistory();
+  const linkHandler = (page: string) => {
+    history.push(page);
+  };
```
Comment (daiHash) [inline src/pages/About/index.tsx:14]: This function its just redirect to a specific page so, I think you it doesn't need to receive a path and add the `/user/myGardens directly`, unless you are going use it to redirect to different pages

Also `let history = useHistory();` => `const`

File: src/pages/Home/home.css [hunk 3]
```diff
@@ -17,13 +17,12 @@
 }
 
 .ni-wa {
-  font-size: 6rem;
-  line-height: 1rem;
+  font-size: 5rem;
   color: #172f4a;
+  line-height: 0%;
 }
```
Comment (daiHash) [inline src/pages/Home/home.css:23]: maybe just `0` without the the `%` is OK

File: src/pages/Home/index.tsx [hunk 4]
```diff
@@ -1,10 +1,17 @@
...
 import { useUserState } from "../../store/user/useUserState";
 import { ReactComponent as Niwa } from "./assets/niwa.svg";
+import Button from '@material-ui/core/Button';
 import "./home.css";
 
 export const Home = () => {
   const { userData } = useUserState();
+  
+  let history = useHistory();
+  const linkHandler = (page: string) => {
```
Comment (daiHash) [inline src/pages/Home/index.tsx:12]: `let history = useHistory();` => `const`

**Review by daiHash — APPROVED** [review]
LGTM!👍
left some comments, please take a look and feel free to merge after resolving the comments

File: src/pages/About/index.tsx (hunk 2 above)
Comment (curtisahuang) [reply src/pages/About/index.tsx:14]: Understood!

File: src/routes.tsx [hunk 5]
```diff
@@ -1,10 +1,14 @@
...
+<<<<<<< HEAD
 import { About } from "./pages/About";
 import { CreateGarden } from "./pages/CreateGarden";
 import { GardenView } from "./pages/GardenView";
 import { Home } from "./pages/Home";
 import { MyGardens } from "./pages/MyGardens";
+=======
+import { Loading } from "./components/LoadingWrapper/Loading";
+>>>>>>> a433826e4fdd0445cc89f3f4b068780e8a94947e
 import { useUserState } from "./store/user/useUserState";
```
Comment (daiHash) [inline src/routes.tsx:12]: @curtisahuang 
Please check the code if all the conflicts are resolved before making a commit and merging your code

**Pull request merged**